Controller/FormWebsiteController.php

```diff
@@ -11,15 +11,21 @@
 
 namespace Sulu\Bundle\FormBundle\Controller;
 
+use Sulu\Bundle\FormBundle\Configuration\FormConfigurationFactory;
+use Sulu\Bundle\FormBundle\Form\HandlerInterface;
 use Sulu\Bundle\FormBundle\Form\Type\AbstractType;
 use Sulu\Bundle\WebsiteBundle\Controller\DefaultController;
 use Sulu\Component\Content\Compat\StructureInterface;
+use Symfony\Component\Form\FormFactoryInterface;
 use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormRegistryInterface;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
 
 class FormWebsiteController extends DefaultController
```

Comment:
@Prokyonn this is a website controller here is not dependency injection used over constructor. This is a ServiceSubscriber which uses the `getSubscribedServices` function to injection services. So you need to extend that function with the services which are not present in the Default Website or AbstractController.